docs/topics/signalen-and-standard-for-public-code.md

```diff
@@ -1,68 +1,189 @@
+There SHOULD be no acronyms, abbreviations, puns or legal/domain specific terms in the codebase without an explanation preceding it or a link to an explanation. | | Glossary needs expanding.
+The name of the project or codebase SHOULD be descriptive and free from acronyms, abbreviations, puns or branding. | yes | SIA --> Signalen
+Documentation SHOULD aim for a lower secondary education reading level, as recommended by the Web Content Accessibility Guidelines. | | Tech documentation will be harder, but a check on signalen.org is needed.
+Any code, documentation and tests MAY have a translation. | |
 
 ## [Use open standards](https://standard.publiccode.net/criteria/open-standards.html)
 
-* Good.
+- [ ]
+
+Requirement | meets | links and notes
+-----|-----|-----
+For features of a codebase that facilitate the exchange of data the codebase MUST use an open standard that meets the Open Source Initiative Open Standard Requirements. | | [StUF-ZKN](https://www.gemmaonline.nl/index.php/Sectormodel_Zaken:_StUF-ZKN) OpenAPI2 spec, GraphQL, Oauth2, OpenID Connect, SMTP, JSON?
+If no existing open standard is available, effort SHOULD be put into developing one. |  | Maybe we need a standard way to forward the complaint between bodies, e.g.: City <--> Provice?
+Standards that are machine testable SHOULD be preferred over those that are not. | N/A |
+Functionality using features from a non-open standard (one that doesn’t meet the Open Source  Initiative Open Standard Requirements) MAY be provided if necessary, but only in addition to compliant features. | |
+All non-compliant standards used MUST be recorded clearly in the documentation. | N/A |
+The codebase SHOULD contain a list of all the standards used with links to where they are available. | no | Created [backend#65](https://github.com/Signalen/backend/issues/65). |
 
 ## [Use continuous integration](https://standard.publiccode.net/criteria/continuous-integration.html)
 
-* Tests not currently invoked by Travis or similar public continuous integration. (Tests are run in the Datapunt CI)
-* Running tests locally is documented in the [README](https://github.com/Amsterdam/signals/blob/master/README.md#running-the-test-suite-and-style-checks)
-* No clear integration testing or capacity testing.
-* Code coverage by tests is not published.
-* Ensure test fixtures are separated from production code.
+- [ ]
+
+Requirement | meets | links and notes
+-----|-----|-----
+All functionality in the source code MUST have automated tests. | mostly | In addition to unit testing, there are end-to-end tests which run on master and release branches (frontend); there are coverage reports, results require manual download.
+Contributions MUST pass all automated tests before they are admitted into the codebase. | yes |
+Contributions MUST be small. | yes | Each pull request contains a single feature or fix, but this policy is not explicit in CONTRIBUTING.md nor explictily checked by reviewer.
```

Comment:
There are some large pieces of functionality that slowed down review by quite a bit. I think we are getting better at breaking up such functionality.